README.md

```diff
@@ -73,24 +73,6 @@ cd ../../..
 
 When the script has finished successfully you can start GOB Management in your browser at: http://localhost:8080
 
-### e2e.sh
```

Comment:
Zouden we hier niet een link kunnen opnemen naar GOB Workflow hoe de e2e tests te starten?